script/ci/prune-cache.sh

```diff
@@ -7,7 +7,7 @@ du -hs target/debug
 
 crate_name="cargo-registry"
 test_name="all"
-bin_names="delete-crate delete-version populate render-readmes server test-pagerduty transfer-crates update-downloads background-worker monitor"
+bin_names="background-worker delete-crate delete-version enqueue-job monitor populate render-readmes server test-pagerduty transfer-crates"
```

Comment:
We could generate this list to avoid having to manually maintain it, e.g.

    cargo metadata --format-version 1 |
        jq -r '.packages[] | select(.name == "cargo-registry") | .targets[] | select(.kind == ["bin"]) | .name'